### PR TITLE
Modify how the table actions are created and implement deleting a row

### PR DESCRIPTION
**Pull request by allysonjp715**
Screen Shot 2020-05-05 at 3 03 32 PM
Screen Shot 2020-05-05 at 3 03 41 PM
Screen Shot 2020-05-05 at 3 03 51 PM

### CODE REVIEW

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 1]
```diff
@@ -41,12 +36,46 @@ interface Case {
...
+            response.then(() => {
+                // Refresh the table data
+                this.state.tableRef.current.onQueryChange();
+            }).catch((e) => {
+                reject(e);
+            });
+        })
+     }
 
+     editCase(newRowData: Case, oldRowData: Case | undefined) {
```
Comment (attwad) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:66]: In which occasion can we edit something undefined?

**Comment by allysonjp715** [comment]
[Create a linelist data deletion mechanism in the curator UI.](https://trello.com/c/VgbursgA/96-create-a-linelist-data-deletion-mechanism-in-the-curator-ui)

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:66]: TBH no idea, but it wouldn't build without that type declaration...

**Comment by allysonjp715** [comment]
Fixed test after merge

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 1 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:66]: ok thanks... let's keep that in mind, we should fail gracefully when we understand when that happens.

**Pull request merged**